### DISCUSSION TITLE
June Personal Web - Create Flag Using Bitmask and Bitwise Operator in Go

### DISCUSSION THREAD

**Discussion by giscus** [ClavinJune/ClavinJune.github.io#15] [General]
# June Personal Web - Create Flag Using Bitmask and Bitwise Operator in Go

create flag using bitmask and bitwise operator in go

https://clavinjune.dev/en/blogs/create-flag-using-bitmask-and-bitwise-operator-in-go/

**Comment by trungdlp-wolffun** [comment]
Hey you
Instead of use
`const (
	PermissionAll = PermissionExecute | PermissionRead | PermissionWrite
)`
you can try:
`const (
	PermissionAll = 0xFF // uint8
)`, this way easy turn on all bit, add new Permission without double check.

**Reply by clavinjune** [reply]
Hi @trungdlp-wolffun thank you for reading!

Yes that's would work in this case. since it is said to be `PermissionAll`, `0xFF` should be okay 👍
